src/super_gradients/training/models/detection_models/yolo_base.py

```diff
@@ -437,27 +437,19 @@ def _check_strides_and_anchors(self):
 
         self.register_buffer('stride', m.stride)  # USED ONLY FOR CONVERSION
 
-    def _initialize_biases(self, cf=None):
+    def _initialize_biases(self):
         """initialize biases into Detect(), cf is class frequency"""
```

Comment:
*DetectX()

Reply:
fixed in https://github.com/Deci-AI/super-gradients/pull/300/commits/075e5c7478335fc969e906fcd7f1a74aec904822